### PR TITLE
[Test] skip test for older validator which check different value.

### PR DESCRIPTION
**Pull request by python3kgae**
Error message was updated in #6196 which cause test fail on older validator.

Since this issue is only caused by error message update, the validation itself is still the same.
The compiler didn't generate anything that behaves badly with older validators.

So just limit these tests to sm6.8+ to make it work on older validator.

### CODE REVIEW

**Comment by damyanp** [comment]
Can you update the description please?  I think there's a bunch of context required to understand why it's ok to just skip this test like this and why we don't need need to test it for older SM's?  We assume that the person reading the commit message it hasn't been following the recent internal chat.

File: tools/clang/unittests/HLSL/ValidationTest.cpp [hunk 1]
```diff
@@ -1057,6 +1059,8 @@ TEST_F(ValidationTest, SimpleHs1Fail) {
       });
 }
 TEST_F(ValidationTest, SimpleHs3Fail) {
+  if (m_ver.SkipDxilVersion(1, 8))
+    return;
```
Comment (pow2clk) [inline tools/clang/unittests/HLSL/ValidationTest.cpp:1063]: SimpleHs1Fail was the test that was altered in #6196. I think this one is fine and that one needs this protection.

Comment (python3kgae) [reply tools/clang/unittests/HLSL/ValidationTest.cpp]: :( Fixed.

**Review by pow2clk — APPROVED** [review]
Thanks for responding to this!

**Review by damyanp — APPROVED** [review]

**Pull request merged**